config-templates/redhat/odf-local/changelog.json

```diff
@@ -3,93 +3,91 @@
         "4.12": {
             "status": "supported",
             "latest-revision": {
+                "revision": "3",
+                "release-date": "17 May 2023",
+                "cves": "CVE-2023-27535, CVE-2022-36227, CVE-2022-3204, CVE-2022-35252, CVE-2022-43552, CVE-2023-29400, CVE-2023-24540,CVE-2023-24539",
+                "golang": "1.19.9",
```

Comment:
Just verify if you have fixed following PSIRTs as well with the new images?
https://github.ibm.com/alchemy-containers/armada-storage/issues/4484
https://github.ibm.com/alchemy-containers/armada-storage/issues/4508
https://github.ibm.com/alchemy-containers/armada-storage/issues/4487
If YES, please add corresponding CVEs as well in the list.

Reply:
Not added